src/components/Book.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Form from './Form';
+import BookList from './BookList';
+
+function Books() {
+  return (
+    <div>
+      <Form />
+      <BookList />
+    </div>
+  );
+}
+
+export default Books;
```

Comment:
- [ OPTIONAL] As required by the previous reviewer  the `Book` component should be the child component of `BookList` and not the other way around, I can suggest you change this in the meantime I will approve the project since it was optional but I can recommend you to follow the structure given by the previous review [link](https://github.com/VTY1999/Bookstore/pull/1/files#r821565478)